Bottom: e933c3c2d604de8a476e6e31608588ee6f2dd67a
Top:    9ecbacaa7ca2dcf06cc27a2d51f31e6fd4047e63
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2009-04-16 14:12:54 -0400

Refresh of stac9766

---

diff --git a/sound/soc/codecs/Kconfig b/sound/soc/codecs/Kconfig
index dc99b51..efec4f9 100644
--- a/sound/soc/codecs/Kconfig
+++ b/sound/soc/codecs/Kconfig
@@ -91,7 +91,7 @@ config SND_SOC_PCM3008
 config SND_SOC_SSM2602
 	tristate
 
-config SND_SOC_9766
+config SND_SOC_STAC9766
 	tristate
 
 config SND_SOC_TAS5504
